internal/infrastructure/http/handlers/handlers.go

```
@@ -0,0 +1,216 @@
+	}
+
+	h.Logger.Info("stored metric", slog.Any("metric", metric))
+
+	bData, err := json.Marshal(metric)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	w.Write(bData)
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)